packages/deployer/package.json

```diff
@@ -14,7 +14,8 @@
     "copy:terraform": "copyfiles terraform/**/**/*.tf dist/",
     "copy:webpack": "copyfiles .webpack/**/**/*.js dist/",
     "prepublishOnly": "yarn build",
-    "webpack": "webpack"
+    "webpack": "webpack",
+    "webpack:dev": "webpack --mode development"
```

Comment:
Where is this script used and what for?

Reply:
It's not used anywhere. I used it manually to get a proper stacktrace in AWS logs. It skips minification so I could see what's actually causing a problem.